mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule GRPC.Mixfile do
     [
       app: :grpc,
       version: @version,
-      elixir: "~> 1.11",
+      elixir: "~> 1.14",
```

Comment:
We still support Elixir 1.11, it's just the formatter that changed a few things in 1.14 :)